lib_nbgl/include/nbgl_use_case_v2.h

```diff
@@ -0,0 +1,162 @@
+    const char *finish_page_text, /* unused on Nano */
+    nbgl_choiceCallback_t choice_callback);
+
+// TODO add an API supporting streaming
+// TODO add an API supporting Skip
+// TODO add an API supporting ReviewLight???
+
+/*
+ * Address verification
+ * This bundles nbgl_useCaseReviewStart / nbgl_useCaseAddressConfirmation
+ */
+
+void nbgl_useCaseAddressReview(
+    const char *address,
+    const nbgl_icon_details_t *icon,
+    const char *reviewTitle,
+    const char *reviewSubTitle,
+    nbgl_choiceCallback_t choice_callback);
+
+// TODO add an API supporting additional tagValueList
+
+
+/*
+ * Status
+ */
+
+// Generic, should not be used
+void nbgl_useCaseStatus(const char *message, bool isSuccess, nbgl_callback_t quitCallback);
+
+
+// For TX, we have the same three variants "transaction", "message" and "operation"
+static inline void nbgl_useCaseTransactionSigned(nbgl_callback_t quitCallback) {
+    nbgl_useCaseStatus("Transaction signed", true, quitCallback);
+}
+
+static inline void nbgl_useCaseTransactionrejected(nbgl_callback_t quitCallback) {
+    nbgl_useCaseStatus("Transaction rejected", false, quitCallback);
+}
+
+static inline void nbgl_useCaseMessageSigned(nbgl_callback_t quitCallback) {
+    nbgl_useCaseStatus("Message signed", true, quitCallback);
+}
+
+static inline void nbgl_useCaseMessageRejected(nbgl_callback_t quitCallback) {
+    nbgl_useCaseStatus("Message rejected", false, quitCallback);
+}
+
+static inline void nbgl_useCaseOperationSigned(nbgl_callback_t quitCallback) {
+    nbgl_useCaseStatus("Operation signed", true, quitCallback);
+}
+
+static inline void nbgl_useCaseOperationRejected(nbgl_callback_t quitCallback) {
+    nbgl_useCaseStatus("Operation rejected", false, quitCallback);
+}
+
+static inline void nbgl_useCaseAddressVerified(nbgl_callback_t quitCallback) {
+    nbgl_useCaseStatus("Address verified", true, quitCallback);
+}
+
+static inline void nbgl_useCaseAddressRejected(nbgl_callback_t quitCallback) {
+    nbgl_useCaseStatus("Address verification\ncancelled", false, quitCallback);
+}
+
+
+/*
+ * Misc
+ */
+
+void nbgl_useCaseChoice(const nbgl_icon_details_t *icon,
+                        const char                *message,
+                        const char                *subMessage,
+                        const char                *confirmText,
+                        const char                *rejectString,
+                        nbgl_choiceCallback_t      callback);
+
+// Same as nbgl_useCaseChoice but as a modal and without icon choice
+void nbgl_useCaseConfirm(const char     *message,
+                         const char     *subMessage,
+                         const char     *confirmText,
+                         const char     *rejectText,
+                         nbgl_callback_t callback);
+
+void nbgl_useCaseSpinner(const char *text);
+
+void nbgl_useCaseViewDetails(const char *tag, const char *value, bool wrapping);
```

Comment:
When is this used ?

Reply:
I just wanted to show that it will be still here...